rust/src/wilcoxon.rs

```diff
@@ -58,7 +67,8 @@ fn main() {
         Ok(_n) => {
             //println!("{} bytes read", n);
             //println!("input:{}", input);
-            const THRESHOLD: usize = 50; // Decrease this number so as to invoke the normal approximation for lower sample sizes. This would speed up the test at the cost of sacrificing accuracy.
+            const MIN_THRESHOLD: usize = 10; // Decrease this number so as to invoke the normal approximation for lower sample sizes. This would speed up the test at the cost of sacrificing accuracy.
+            const MAX_THRESHOLD: usize = 1000;
```

Comment:
Why is it necessary to specify a max threshold?

Reply:
If one vector is super large (say 1 million) and the other is 2. The number of combinations will still be very high. In this case, the normal approximation will be used.